src/content/smart-bulletins/030.md

```diff
@@ -0,0 +1,88 @@
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| Dec 18, 2019 | Implementation of Smart Bulletin #30. |
+| August 28, 2023 | Revised to update for administrative change and also replaces Smart Bulletin #21. |
+
+## Effective Date
+
+Effective upon issuance and shall remain in force until modified,
+rescinded, or superseded
+
+## Business Line(s) Affected
+
+Purchase, Travel, Fleet and Integrated
+
+## Introduction
+
+The purpose of this GSA SmartPay Smart Bulletin is to provide agencies/organizations with implementing guidance on submitting statistical and narrative reports as required in Chapter 5.3 of Office of Management and Budget (OMB) Circular A-123, Appendix B, entitled “A Risk Framework for Government Charge Card Programs,” dated August 27, 2019. This Smart Bulletin includes: (1) reporting templates for agency submission of chapter 5.3 reports,(2) information on a due date extension solely for FY2019 reporting, and (3) reporting expectations for FY2020 and beyond.
+
+## Summary
+
+OMB memorandum M-17-26 dated June 25, 2017 discontinued agency/organization purchase and travel charge card statistical reporting to OMB. However, OMB in that memorandum also required agencies to maintain that statistical reporting information for their own use in the management of their charge card programs. It further designated GSA’s Center for Charge Card Management (CCCM) as the central repository for the agency charge card statistical data which was previously required to be submitted to OMB. CCCM will post this statistical and narrative data on GSA Interact for agencies/organizations subject to the Chief Financial Officers’ (CFO) Act. CCCM may also post information pertaining to other GSA SmartPay participating agencies/organizations as well. CCCM will work with the GSA SmartPay contractor banks to obtain agency/organization statistical reporting data in an electronic, automated manner to the extent possible. The balance of the statistical data CCCM is unable to directly obtain from contractor banks will need to be provided by the CFO Act agencies/organizations (and other entities as required) to CCCM. CCCM will use a central repository for these reports. The Circular also states CCCM “may issue further guidance on both narrative and statistical reporting through the GSA Smart Bulletin process.” This bulletin constitutes that further guidance.
+
+### OMB Circular A-123, Appendix B Chapter 5.3.1 – Statistical Reporting –
+Requested information in the template attached below shall be submitted, at a minimum, on an annual fiscal year basis. Agency-provided information shall be furnished to CCCM *no later than 90 calendar days after the end of the Fiscal Year (FY)*. The due date for agency/organization submission of FY2019 agency statistical reporting to CCCM is hereby extended to No Later Than (NLT) February 28, 2020. This extension applies only to the FY 19 reports.
```

Comment:
The requested information in the template attached below shall be submitted, at a minimum, on an annual fiscal year basis. Agency-provided information shall be furnished to CCCM no later than 90 calendar days after the end of the Fiscal Year (FY). The due date for agency/organization submission of FY2019 agency statistical reporting to CCCM is hereby extended to No Later Than (NLT) February 28, 2020. This extension applies only to the FY 19 reports.
The Statistical Reporting requirement applies to all business lines under the GSA SmartPay program. Agencies must report the following items for each business line:
Count of accounts, as defined by GSA;
Total number of accounts across the charge card program as compared to the prior reporting period;
Total charge card dollars spent; and
Total refunds earned.
Specific requirements for each business line are as follows:
Purchase cards only (to be provided to CCCM by CFO Act Agencies)
Ratio of A/Os to purchase cardholders (span of control);
Number of purchase cards with transaction limits over the micro-purchase threshold
Ratio of the number of confirmed violations reported pursuant to P.L. 112-194 as compared to the number of valid transactions within the reporting period (GSA’s Center for Charge Card Management will obtain confirmed violation information from OMB/OFFM).
Convenience checks only:
Total number of convenience checks written;
Number of checks written over micro-purchase threshold;
Number of checks written under micro-purchase threshold; and
Total dollar amount of checks written.
Fleet cards only (to be provided to CCCM by CFO Act Agencies):
Ratio of reviewing officials to fleet cards (span of control); and
Number of fleet cards with transaction limits over micro-purchase threshold. Integrated cards only (to be provided to CCCM by CFO Act Agencies):
Ratio of A/Os to integrated cardholders (purchase business line - span of control);
Ratio of reviewing officials to integrated cards (fleet business line - span of control);
Number of integrated cards with transaction limits over the micro-purchase threshold and
Ratio of the number of confirmed violations reported pursuant to P.L. 112-194 as compared to the number of valid transactions within the reporting period (GSA’s Center for Charge Card Management will obtain confirmed violation information from OMB/OFFM).